xrootd.sh

```diff
@@ -5,19 +5,25 @@ source: https://github.com/xrootd/xrootd
 requires:
  - "OpenSSL:(?!osx)"
  - Python-modules:(?!osx_arm64)
- - AliEn-Runtime
  - libxml2
+ - AliEn-Runtime
 build_requires:
  - CMake
  - "osx-system-openssl:(osx.*)"
  - "GCC-Toolchain:(?!osx)"
  - UUID:(?!osx)
  - alibuild-recipe-tools
 prepend_path:
-  PYTHONPATH: "$XROOTD_ROOT/lib/python/site-packages"
+  PYTHONPATH: "${XROOTD_ROOT}/lib/python/site-packages"
 ---
 #!/bin/bash -e
-[[ -e $SOURCEDIR/bindings ]] && { XROOTD_V4=True; XROOTD_PYTHON=True; } || XROOTD_PYTHON=False
+
+if [[ $ALIEN_RUNTIME_VERSION ]]; then
+  # AliEn-Runtime: we take libxml2 from there, in case they were not taken from the system
+  LIBXML2_ROOT=${LIBXML2_REVISION:+$ALIEN_RUNTIME_ROOT}
+fi
+
+[[ -e ${SOURCEDIR}/bindings ]] && { XROOTD_V4=True; XROOTD_PYTHON=True; } || XROOTD_PYTHON=False
```

Comment:
As the `cmake` command below uses `${XROOTD_PYTHON:+some option here}`, you should probably unset `XROOTD_PYTHON`, instead of setting it to `False`. If you set `XROOTD_PYTHON=False`, XRootD will always build with `-DENABLE_PYTHON=ON`.